Cargo.toml

```diff
@@ -19,7 +19,9 @@ slug = "=0.1.1"
 env_logger = "0.7"
 r2d2 = "0.8"
 r2d2_postgres = "0.14"
-url = "1.4"
+# iron needs url@1, but it reexports it as iron::url, so we can start using
+# url@2 for other usecases
+url = { version = "2.1.1", features = ["serde"] }
```

Comment:
Why add a second version of url2?

Reply:
I assume that's "why _not_ add"?

All existing use of `url` was directly related to `iron`, and they happen to reexport it; so it seemed less confusing to use `iron::url` and `url` instead of `url` and `url2`.

Reply:
Right, but why use version 2 of url at all?

Reply:
`serde 1.*` compatibility, `url 1.*` is stuck back on `serde >=0.6.1, <0.9`